kuma/api/v1/urls.py

```diff
@@ -22,6 +22,15 @@
         plus.landing_page_survey,
         name="api.v1.plus.landing_page_survey",
     ),
-    path("subscriptions/", views.subscriptions, name="api.v1.subscriptions"),
+    path(
+        "subscriptions/checkout",
```

Comment:
```suggestion
        "subscriptions/checkout/",
```
I've thought a lot about this and come to conclude that trailing slashes are better. 
Yes, there are some endpoints above that don't use a trailing slash and that's unfortunate and something we should eventually fix, too. But all the new `landing-page` related endpoints all have trailing slashes. They were easy because there was 0 need do worry about parity about old kuma. 
If you agree and go for this, we're going to need to remember to update yari too but it's not time-critical. 

The advantages with trailing slashes are:
- If you failed to include the `/`, Django has well-tested redirects that can save you. But you can't do that when the trailing slash is *not* there. For example `https://developer.mozilla.org/api/v1/settings/` will 404, but if line 7 (above) had a trailing slash both `https://developer.mozilla.org/api/v1/settings/` and `https://developer.mozilla.org/api/v1/settings` would work. `fetch` would just follow the redirect. 
- If you decide to extend the URL, it feels more right to build on. E.g. someday we decide to add `/subscriptions/checkout/ping/`.
- Trailing slashes is how URLs are supposed to work and the only reason for not using it is vanity in the address bar or when you put a link in an email or in some marketing. The vanity argument doesn't make sense of an API. 

I can think of counter-arguments too. And therefore I'm pretty soft in my conviction. But I would definitely appreciate a discussion first and foremost.

Reply:
I see your points and I don't have recent experiences pushing me in either direction so I'm going to go with your good arguments.